src/openms/include/OpenMS/ANALYSIS/ID/ConsensusIDAlgorithmPEPMatrix.h

```diff
@@ -160,31 +54,19 @@ namespace OpenMS
   public:
     /// Default constructor
     ConsensusIDAlgorithmPEPMatrix();
-
+    /// Sequence similarity based on substitution matrix (ignores PTMs)
+    double getSimilarity_(AASequence seq1, AASequence seq2) override; //danach wieder zu private machen (nur zum Testen)
```

Comment:
you can make the method `protected:`, and then use it in the test-class by simply deriving from it:
```
NeedmanWunschTest : public NeedmanWunsch
{
  double getSim(...) { return getSimilarity_(...); };
};
```
This way, you can test the protected function.

Reply:
`const aaseq&`